### ISSUE TITLE
Mute Minehut Tutorials

### ISSUE THREAD

**Issue by MrScopes** [Minehut/Meta#304]
### Feature request

**Checklist**
- I have read and agree to the README (found on https://github.com/Minehut/Meta)
- This is not a bug report, support request or plugin request
- I have searched the Issue Tracker to make sure this is not a duplicate issue
- This issue does not contain any sensitive information 

**Platform**
Website

**Describe the feature**
The youtube videos should be muted by default whenever you're starting your server.

**How the feature is useful**
Sometimes it's sort've startling lol, would be nice if it was muted by default

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by LukasmanMHdude** [comment]
It would be nice if they were removed

**Comment by farwl** [comment]
^ It would be nice if they were removed

**Comment by MrScopes** [comment]
yeah honestly an option to just disable them entirely would be nice

**Comment by drtshock** [comment]
Hey running these videos is how we pay to run all the free servers.

That being said, if you mute the player, does it stay muted for you next time? I think that might be a more reasonable compromise.

**Comment by LukasmanMHdude** [comment]
Can we get rid of the transferring for paid servers at least?

**Comment by drtshock** [comment]
Yeah that's a more reasonable option for us.

**Comment by MrScopes** [comment]
> that being said, if you mute the player, does it stay muted for you next time? I think that might be a more reasonable compromise.

ideally

**Comment by MiginsMC** [comment]
It already does that for me, unless that's a chrome thing
